rascal-vscode-extension/src/test/vscode-suite/ide.test.ts

```diff
@@ -125,6 +125,11 @@ describe('IDE', function () {
         await triggerTypeChecker(editor, TestWorkspace.mainFileTpl, true);
     });
 
+    it("type checker runs on dependencies", async() => {
+        const editor = await ide.openModule(TestWorkspace.libCallFile);
```

Comment:
Doesn't this test depend on the state of previous checks? shouldn't we make sure to delete some tpl files first?

Reply:
The line that follows this one specifies the `tpl` file of the lib file (i.e., the file being called). This `tpl` file is deleted by the test framework before starting the test.

Reply:
Ok, nice